main.js

```diff
@@ -0,0 +1,30 @@
+// Get array data from api to variable
+function fetchData(cb, func) {
+  $.ajax({
+    url: "https://randomuser.me/api/?results=15",
+    success: function (response) {
+      usersArr = response
+      cb(usersArr, func)
```

Comment:
Not sure why you are using so many callbacks. Actually, callbacks are confusing for humans to read, and we will learn later on how we can minimize using them. Here they seem unnecessary so I would drop them.